opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -518,6 +518,28 @@ message DoubleHistogramDataPoint {
   // (Optional) List of exemplars collected from
   // measurements that were used to form the data point
   repeated DoubleExemplar exemplars = 8;
+
+  // Note: the definition used to for min and max fields below is
+  // based on temporality in a way meant to match the Prometheus
+  // Summary data type used with quantiles 0 and 1, respectively, when
+  // paired with cumulative temporality.  This uses a fixed width that
+  // SHOULD be up to 5-10 minutes or for as long as data is available.
+  // https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#summary
+  
+  // min is the minimum value over a recent window of time.
```

Comment:
Nit: Declare min/max as (Optional) in the docs?

Reply:
As discussed in the metrics SIG, Min and max can be made optional by adding flags to [DataPointFlags](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto#L300-L316).

I propose:
```
  // This HistogramDataPoint contains no recorded min value.  
  // This value SHOULD be used to reflect the intentional omission 
  // of the optional histogram min field, or that no values were 
  // recorded in the window of time.
  FLAG_HISTOGRAM_NO_MIN_VALUE = 2;

  // This HistogramDataPoint contains no recorded max value.  
  // This value SHOULD be used to reflect the intentional omission 
  // of the optional histogram max field, or that no values were 
  // recorded in the window of time.
  FLAG_HISTOGRAM_NO_MIN_VALUE = 2;
```

Reply:
cc: @jsuereth mentioned the idea of using field presence for this, which I think I would prefer (if we can agree).

Reply:
Like it!